packages/blue-migration-sdk-viem/src/fetchers/compoundV3/compoundV3.fetchers.ts

```diff
@@ -0,0 +1,174 @@
+import { type Address, ChainUtils } from "@morpho-org/blue-sdk";
+import { isDefined, values } from "@morpho-org/morpho-ts";
+
+import type { FetchParameters } from "@morpho-org/blue-sdk-viem";
+import type { Client } from "viem";
+import { getChainId, readContract } from "viem/actions";
+import { cometAbi, cometExtAbi } from "../../abis/compoundV3.abis.js";
+import MIGRATION_ADDRESSES from "../../config.js";
+import type { MigratablePosition } from "../../positions/index.js";
+import { MigratableSupplyPosition_CompoundV3 } from "../../positions/supply/compoundV3.supply.js";
+import {
+  MigratableProtocol,
+  SupplyMigrationLimiter,
+} from "../../types/index.js";
+import { rateToApy } from "../../utils/rates.js";
+
+async function fetchCompoundV3InstancePosition(
```

Comment:
```suggestion
async function fetchCompoundV3Position(
```

Reply:
I don't like to have two functions named `fetchCompoundV3Positions` and `fetchCompoundV3Position` in the same context :/

Reply:
it can easily create confusion imo